bitcoin/src/blockdata/weight.rs

```diff
@@ -120,13 +120,89 @@ mod tests {
     use super::*;
 
     #[test]
-    fn checked_sub_test(){
+    fn weight_constructor_test() {
+        assert_eq!(Weight::ZERO, Weight::from_wu(0));
+    }
+
+    #[test]
+    fn from_vb_test() {
+        let vb = Weight::from_vb(1).expect("expected weight unit");
+        assert_eq!(Weight(4), vb);
```

Comment:
FYI, in general it's more useful to test API rather than internal representation. So calling `assert_eq!(vb.to_wu(), 4)` would be a better test. This is for the case we decide to change internal representation.

However since this is very unlikely, it's OK here.

Reply:
Point taken.  I'm fine to change it to that if you'd like.  Although since you've acked the commit I'll leave it for now.